docs/source/index.md

```diff
@@ -2,27 +2,27 @@
 
 ```{toctree}
 :hidden:
-standard.md
+specification.md
 evaluation.md
 contributing.md
 contributors.md
 ```
 
 ## Introduction
 
-Motivation: Why a TRE standard is needed/ useful and a description of the broader SATRE project, conception and goals
+Motivation: Why a TRE specification is needed/ useful and a description of the broader SATRE project, conception and goals
```

Comment:
Is this placeholder text?

(Just a question, let's not fix this in this PR)

Reply:
Yes. Moved into a comment now.